CodePush.js

```diff
@@ -237,7 +240,7 @@ const sync = (() => {
   let syncInProgress = false;
   const setSyncCompleted = () => { syncInProgress = false; };
 
-  return (options = {}, syncStatusChangeCallback, downloadProgressCallback) => {
+  return (options = {}, syncStatusChangeCallback, downloadProgressCallback, handleBinaryVersionMismatch) => {
```

Comment:
If this 4th parameter is callback may be it make sense to add *callback* postfix to this to avoid confusion (on first sight looks like boolean variable for me, not a function)